contracts/src/v0.8/automation/dev/interfaces/v2_2/IChainModule.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.19;
```

Comment:
nitpick, but we can probably unlock this to ^0.8.0

Reply:
done